ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/adoptopenjdk10/tasks/main.yml

```diff
@@ -21,7 +21,9 @@
   shell: ls -ld /usr/lib/jvm/jdk-10* >/dev/null 2>&1
   ignore_errors: yes
   register: adoptopenjdk10_installed
-  tags: adoptopenjdk10
+  tags:
+    - adoptopenjdk10
+    - skip_ansible_lint
```

Comment:
Does that mean skip the linter?

Reply:
I'm assuming so ... I just copied it from the adoptopenjdk9 one ... As mentioned elswehere recently we need to try and implement some ansible best practices instead of doing this stuff but that should probably be part of a general overhaul

Reply:
yeah - we shouldn't skip the linting really :-)